packages/app-project/src/shared/components/SubjectPicker/SubjectPicker.spec.js

```diff
@@ -92,11 +36,11 @@ describe('Components > Subject Picker', function () {
     expect(columnHeadings).to.exist()
   })
 
-  it('should have a row for each subject', function () {
+  it.skip('should have a row for each subject', function () {
```

Comment:
The mock request that populates the table doesn't seem to be running in JSDOM. It does run when the story is viewed in a browser.

Reply:
Turns out that requests using `superagent` still need `nock` in Node. I'm not sure why.

Reply:
Anyhow, this test isn't skipped any more, and is working as expected.